scripts/tools/zap/tests/outputs/all-clusters-app/app-templates/endpoint_config.h

```diff
@@ -302,15 +302,15 @@
             { (uint16_t) 0x0, (uint16_t) 0x0, (uint16_t) 0x7 }, /* ControlMode */                                                  \
                                                                                                                                    \
             /* Endpoint: 1, Cluster: Thermostat (server) */                                                                        \
-            { (uint16_t) 0xA28, (uint16_t) -0x6AB3, (uint16_t) 0x7FFF }, /* OccupiedCoolingSetpoint */                             \
```

Comment:
I assume this is due to clang-format getting updated?

Reply:
Yes, this is unfortunate :(